angular/src/app/modules/admin/cvstatus/cvstatus.component.ts

```
@@ -0,0 +1,76 @@
+import { MatDialog } from '@angular/material/dialog';
+import { PagedListingComponentBase, PagedRequestDto } from '@shared/paged-listing-component-base';
+import { CvstatusService } from '../../../service/api/cvstatus.service';
+import { catchError, finalize } from 'rxjs/operators';
+
+@Component({
+  selector: 'app-cvstatus',
+  templateUrl: './cvstatus.component.html',
+  styleUrls: ['./cvstatus.component.css']
+})
+export class CVStatusComponent extends PagedListingComponentBase<CVStatusComponent> implements OnInit {
+  protected list(request: PagedRequestDto, pageNumber: number, finishedCallback: Function): void {
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)